.pipelines/ESRPSigning_core.json

```diff
@@ -51,6 +51,8 @@
                 "modules\\FileExplorerPreview\\PowerToys.ManagedTelemetry.dll",
```

Comment:
Offhand, this file should probably be added to `excludes.txt`:
```
^\Q.pipelines/ESRPSigning_core.json\E$
```

I mention this because locally, I get:

```
.pipelines/ESRPSigning_core.json: line 189, columns 97-103, Warning - `codicon` is not a recognized word. (unrecognized-spelling)
.pipelines/ESRPSigning_core.json: line 189, columns 87-94, Warning - `codicons` is not a recognized word. (unrecognized-spelling)
installer/PowerToysSetup/PowerToys.wxs: line 94, columns 40-42, Warning - `CCB` is not a recognized word. (unrecognized-spelling)
installer/PowerToysSetup/PowerToys.wxs: line 94, columns 30-38, Warning - `CFFDAFADA` is not a recognized word. (unrecognized-spelling)
installer/PowerToysSetup/PowerToysInstaller.wixproj: line 106, columns 7-18, Warning - `Autogenerate` is not a recognized word. (unrecognized-spelling)
.pipelines/ESRPSigning_core.json: line 189, columns 106-112, Warning - `codicon` is not a recognized word. (unrecognized-spelling)
```

And, afaict, there's nothing in this file worth checking.

Reply:
newb q, how do we exclude files in excludes.txt?

Reply:
Sorry if my comment was confusing, the first block was the text to add.

That form is an exact path exclusion - the `^\Q`...`\E$` decorations cause the inside to be treated as raw and matched from the beginning to the end of the path.

For directories, or patterns you can use other forms. - I think there are enough examples in the file, but if not, I'd love to talk through hour to make it easier for people to understand how to use it. (Readme in the directory, advice file, wiki)

Reply:
Moving to issue: 15753